packages/react-components/src/helpers/utilities.js

```diff
@@ -0,0 +1,48 @@
+export const uniqueId = (
+  counter =>
+  (str = '') =>
+    `${str}${++counter}`
+)(0);
+
+export const isUndefined = val => val === undefined;
```

Comment:
Since this is so simple and we're only using it once I think it can just be inlined.